ops/aws/canary/lambda/pkg/scenarios/utils.go

```diff
@@ -131,16 +133,13 @@ func compareOutput(output []byte, expectedOutput string) error {
 }
 
 func getClient() *publicapi.RequesterAPIClient {
-	apiHost := config.GetAPIHost()
-	apiPort := config.GetAPIPort()
-	if apiHost == "" {
-		apiHost = system.Envs[system.GetEnvironment()].APIHost
-	}
-	if apiPort == nil {
-		defaultPort := system.Envs[system.GetEnvironment()].APIPort
-		apiPort = &defaultPort
+	hostStr := os.Getenv("BACALHAU_HOST")
+	portStr := os.Getenv("BACALHAU_PORT")
+	apiport, err := strconv.ParseInt(portStr, 10, 64)
+	if err != nil {
+		panic(err)
 	}
-	return publicapi.NewRequesterAPIClient(apiHost, *apiPort)
+	return publicapi.NewRequesterAPIClient(hostStr, uint16(apiport))
```

Comment:
Hm, I seem to remember that this logic was important for allowing the canary tests to pass in CI as well as having a functioning canary in production. @wdbaruni do we set these env variables in AWS?

Reply:
We are passing `BACALHAU_ENVIRONMENT` (e.g. `staging` or `production`) to the canary, which then uses the default host and port of that environment ([here](https://github.com/bacalhau-project/bacalhau/blob/main/ops/aws/canary/lib/canary-stack.ts#L109))

I can help with migrating the canary either after this PR is merged or I can add a commit. Basically we need to add the host and port in [cdk.json](https://github.com/bacalhau-project/bacalhau/blob/main/ops/aws/canary/cdk.json), and pass those to the lambda's environment

Reply:
I would _love_ this to be landed in a follow on if required.

Reply:
yeah that is fine. just create a follow up task and assign it to me